src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -272,7 +272,7 @@ public void licenseHeaderFile(Object licenseHeaderFile, String delimiter) {
 	/** Sets up a FormatTask according to the values in this extension. */
 	protected void setupTask(FormatTask task) throws Exception {
 		task.paddedCell = paddedCell;
-		task.lineEndingsPolicy = getLineEndingPolicy();
+		task.lineEndingPolicy = getLineEndingPolicy();
```

Comment:
I don't care which name we use, but this is a public member of a public class documented in the README, so changing it breaks API compat for everyday buildscripts.  These changes are likely going to require a major version bump anyway, but I'd rather not churn the `lineEndingsPolicy` name.

aside: I've enjoyed learning about a lot of Stream methods I didn't know about before, lots of good cleanup in various parts 👍